appveyor.yml

```diff
@@ -44,16 +64,22 @@ environment:
       CONDA_PY: 36
       CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
 
+    - TARGET_ARCH: x86
+      CONDA_NPY: 112
+      CONDA_PY: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
+
+    - TARGET_ARCH: x64
+      CONDA_NPY: 112
+      CONDA_PY: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
+
 
 # We always use a 64-bit machine, but can build x86 distributions
 # with the TARGET_ARCH variable.
 platform:
     - x64
 
-# Build on this OS until we can figure out how to workaround a compiler bug
-# introduced in Update 3.
-os: Visual Studio 2015 Update 2
```

Comment:
There has been a workaround in this repo that you also have in astropy. I deleted it with the update.

@conda-forge/core Is is OK if I add the workaround in the appveyor file again?